src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -76,45 +61,7 @@ def execute_tests(self) -> TestComponentResults:
         pass
 
     def multi_execute_integtest_sh(self, cluster_endpoints: list, security: bool, test_config: str) -> int:
```

Comment:
Make this method abstract. So that is its implemented whoever extends this suite.

Reply:
Sure Sayali